2021-11-09 16:29:32,064 INFO    MainThread:25488 [wandb_setup.py:_flush():69] setting env: {}
2021-11-09 16:29:32,065 INFO    MainThread:25488 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-09 16:29:32,065 INFO    MainThread:25488 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_162932-1jb8xati\logs\debug.log
2021-11-09 16:29:32,065 INFO    MainThread:25488 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_162932-1jb8xati\logs\debug-internal.log
2021-11-09 16:29:32,065 INFO    MainThread:25488 [wandb_init.py:init():375] calling init triggers
2021-11-09 16:29:32,066 INFO    MainThread:25488 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-11-09 16:29:32,066 INFO    MainThread:25488 [wandb_init.py:init():424] starting backend
2021-11-09 16:29:32,066 INFO    MainThread:25488 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-11-09 16:29:32,071 INFO    MainThread:25488 [backend.py:ensure_launched():135] starting backend process...
2021-11-09 16:29:32,167 INFO    MainThread:25488 [backend.py:ensure_launched():139] started backend process with pid: 26324
2021-11-09 16:29:32,167 INFO    MainThread:25488 [wandb_init.py:init():429] backend started and connected
2021-11-09 16:29:32,168 INFO    MainThread:25488 [wandb_init.py:init():477] updated telemetry
2021-11-09 16:29:32,168 INFO    MainThread:25488 [wandb_init.py:init():500] communicating current version
2021-11-09 16:29:33,499 INFO    MainThread:25488 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-09 16:29:33,499 INFO    MainThread:25488 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-11-09 16:29:33,753 INFO    MainThread:25488 [wandb_init.py:init():540] starting run threads in backend
2021-11-09 16:29:38,199 INFO    MainThread:25488 [wandb_run.py:_console_start():1601] atexit reg
2021-11-09 16:29:38,200 INFO    MainThread:25488 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-11-09 16:29:38,200 INFO    MainThread:25488 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-11-09 16:29:38,202 INFO    MainThread:25488 [wandb_run.py:_redirect():1536] Redirects installed.
2021-11-09 16:29:38,202 INFO    MainThread:25488 [wandb_init.py:init():565] run started, returning control to user process
2021-11-09 16:29:38,202 INFO    MainThread:25488 [wandb_config.py:__setitem__():141] config set data = {'name': 'kaggle_houses', 'n_jobs': -1} - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000016A63537280>>
2021-11-09 16:29:38,202 INFO    MainThread:25488 [wandb_run.py:_config_callback():843] config_cb data {'name': 'kaggle_houses', 'n_jobs': -1} None
2021-11-09 16:29:42,625 INFO    MainThread:25488 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-11-09 16:29:42,626 INFO    MainThread:25488 [wandb_run.py:_restore():1543] restore
2021-11-09 16:29:43,276 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 978
  total_bytes: 978
}

2021-11-09 16:29:43,453 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 978
  total_bytes: 978
}

2021-11-09 16:29:43,867 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 978
  total_bytes: 13716
}

2021-11-09 16:29:43,984 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 978
  total_bytes: 14042
}

2021-11-09 16:29:44,090 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,201 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,314 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,421 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,532 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,663 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:44,771 INFO    MainThread:25488 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14042
  total_bytes: 14042
}

2021-11-09 16:29:45,892 INFO    MainThread:25488 [wandb_run.py:_show_summary():1848] rendering summary
2021-11-09 16:29:45,897 INFO    MainThread:25488 [wandb_run.py:_show_files():1915] logging synced files
